src/pages/LeeSaemi/Login/Login.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import './Login.scss';
+
+const Login = () => {
+  const [inputId, setInputId] = useState('');
+  const [inputPwd, setInputPwd] = useState('');
+  const saveUserId = event => {
+    setInputId(event.target.value);
+  };
+
+  const saveUserPwd = event => {
+    setInputPwd(event.target.value);
+  };
+
+  const isInputValid = inputId.includes('@') && inputPwd.length >= 5;
+
+  const goToMain = () => {
+    fetch('http://10.58.52.162:8000/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: inputId,
+        password: inputPwd,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        if (data.message === '로그인 성공') {
+          localStorage.setItem('jwtToken', data.jwtToken);
+        } else if (data.message === '비밀번호가 틀렸습니다.') {
+          alert('사용자의 아이디 혹은 비밀번호가 틀렸습니다');
+        } else if (data.message === '존재하지 않는 이메일입니다.') {
+          alert('존재하지 않는 사용자입니다.');
+        }
```

Comment:
다양한 메세지에 대한 예외처리를 해주는것도 좋지만 앞선 경우와 하나도 일치하지 않을 경우에 대한 예외처리도 진행되면 좋겠습니다!
```suggestion
        if (data.message === '로그인 성공') {
          localStorage.setItem('jwtToken', data.jwtToken);
        } else if (data.message === '비밀번호가 틀렸습니다.') {
          alert('사용자의 아이디 혹은 비밀번호가 틀렸습니다');
        } else if (data.message === '존재하지 않는 이메일입니다.') {
          alert('존재하지 않는 사용자입니다.');
        } else{
          alert('로그인을 다시 시도해주세요.');        
        }
```